.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 365
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: false
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had recent activity. 
+  It will be reviewed by the core Carbon team and may be closed if no further activity occurs. Thank you for your contributions.
```

Comment:
Perhaps need to add here that if someone doesn't think it's stale, then they should add a comment

Reply:
updated